### PR TITLE
Allow to specify a target path while generating new app

### PR DESCRIPTION
**Pull request by psantos10**
With this I can be at `/Users/psantos/Desktop` location, and run the following:

> $ hanami new bookshelf --path=/Users/psantos/Workspace/learning/learning_ruby_and_hanami

And my application will be generated at this location: `/Users/psantos/Workspace/learning/learning_ruby_and_hanami`

### CODE REVIEW

File: lib/hanami/commands/new/abstract.rb [hunk 1]
```diff
@@ -35,8 +35,10 @@ def initialize(options, name)
         end
 
         def start
-          FileUtils.mkdir_p(app_name)
-          Dir.chdir(app_name) do
+          path = options[:path] || '.'
+          application_name = "#{path}/#{app_name}"
```
Comment (Erol) [inline lib/hanami/commands/new/abstract.rb]: Might be better to use `File.join` 🐹

Comment (Erol) [inline lib/hanami/commands/new/abstract.rb]: Might be better to use `File.join` 🐹

**Comment by psantos10** [comment]
What about now @Erol ?

File: lib/hanami/commands/new/abstract.rb [hunk 2]
```diff
@@ -35,10 +35,12 @@ def initialize(options, name)
...
         def start
-          FileUtils.mkdir_p(app_name)
-          Dir.chdir(app_name) do
+          path = options[:path] || '.'
+          application_name = File.join(path, app_name)
+          FileUtils.mkdir_p(application_name)
+          Dir.chdir(application_name) do
             @target_path = Pathname.pwd
-
+
```
Comment (davydovanton) [inline lib/hanami/commands/new/abstract.rb]: Remove this line please ❤️

Comment (psantos10) [inline lib/hanami/commands/new/abstract.rb]: @davydovanton you are talking about the line 43 or 42?

Comment (davydovanton) [inline lib/hanami/commands/new/abstract.rb]: I'm talking about empty line with blank characters (43) 😃

**Comment by jodosha** [comment]
@psantos10 Thanks for this PR, but I'm refuse to include features that can be easily solved with a UNIX command:

```shell
mkdir -p Code/learning && cd $_ # this takes you to /Users/luca/Code/learning
hanami new bookshelf
```

**Comment by psantos10** [comment]
Hi @jodosha 

But installing this way, which `hanami` version will be used? installed by gem or the one I have clone locally?

**Comment by jodosha** [comment]
@psantos10 The one from the gem. To use the local cloned repo:

```shell
cd Code/hanami/hanami
bundle exec hanami new bookshelf --hanami-head=true
cd bookshelf
vim Gemfile # Change gem 'hanami', github: 'hanami/hanami'  into gem 'hanami', path: '..'
bundle
```

In case you want to move `bookshelf` somewhere else then `:path` option must correspond to the path to the Hanami cloned repository. In my case `/Users/luca/Code/hanami/hanami`.

**Pull request closed without merging**